test-utils/testlib/src/actix_utils.rs

```diff
@@ -32,6 +34,8 @@ impl ShutdownableThread {
 impl Drop for ShutdownableThread {
     fn drop(&mut self) {
         self.shutdown();
+        // Leaving some time for all threads to stop after system is stopped.
+        thread::sleep(Duration::from_millis(100));
```

Comment:
It seems that it is not a solution, but a symptomatic patch. Is there an issue about this?

Reply:
I agree we should be explicitly waiting for all threads to stop.